e2e/dragonfly_controller_test.go

```diff
@@ -40,7 +40,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-var _ = Describe("Dragonfly Lifecycle tests", Ordered, func() {
+var _ = Describe("Dragonfly Lifecycle tests", Ordered, FlakeAttempts(3), func() {
```

Comment:
What does FlakeAttempts() do? Try it 3 times?

Reply:
Yep!